txscript/stack.go

```diff
@@ -204,11 +204,6 @@ func (s *stack) Tuck() error {
 // DropN(1): [... x1 x2] -> [... x1]
 // DropN(2): [... x1 x2] -> [...]
 func (s *stack) DropN(n int32) error {
-	if n < 1 {
```

Comment:
Since this is no longer returning an error for `DropN(0)` I think it'd be good to add an example in the comments above showing the expected `NOP` behaviour, for reference purposes.

Reply:
Good suggestion.  Updated.